src/server/database/seeds/test/users.js

```diff
@@ -0,0 +1,14 @@
+import User from '@models/User'
+
+const TEST_ACCOUNTS = [
+  {
+    name: 'Email in use account',
+    email: 'email@inuse.com',
+    passwordPlain: 'EMAIL_IN_USE_ACCOUNT'
+  }
+]
+
+export const seed = async knex => {
+  await knex('users').truncate()
+  await Promise.all(TEST_ACCOUNTS.map(User.createUser))
+}
```

Comment:
I think keeping the seed files separate in each environment by model/table is the way to go.